.github/workflows/e2e-nvidia-t4-x1.yml

```diff
@@ -95,17 +95,18 @@ jobs:
       - name: Fetch and checkout PR
         id: fetch_pr
         if: github.event_name == 'pull_request_target'
+        working-directory: ./eval
         run: |
           git fetch origin pull/${{ github.event.pull_request.number }}/head:pr-${{ github.event.pull_request.number }}
           git checkout pr-${{ github.event.pull_request.number }}
 
-      - name: Install system packages
+      - name: Install Packages
```

Comment:
I'd prefer to keep this as is

Reply:
I was trying to match the state of the training repo and missed.  Fixed!